src/core/searchUtils.js

```diff
@@ -4,6 +4,7 @@ import defaultConfig from 'config';
 import log from 'core/logger';
 
 export const operatingSystems = {
+  Ubuntu: 'linux',
```

Comment:
Is this required?

Reply:
It's taking Ubuntu as a different OS, so did this to redirect it to linux.

Reply:
Ok, I see. I don't know if we want to expand this list or how, but why not.